workflows/genomes_generation.nf

```diff
@@ -115,15 +115,23 @@ workflow GGP {
     FASTQC_BEFORE (assembly_and_runs.map{ meta, _ , reads -> [meta, reads] })
     ch_versions = ch_versions.mix(FASTQC_BEFORE.out.versions)
 
-    // ---- pre-processing ---- //
-    PROCESS_INPUT( assembly_and_runs ) // output: [ meta, assembly, [raw_reads] ]
-
-    ch_versions = ch_versions.mix( PROCESS_INPUT.out.versions )
+    def input_assemblies_and_runs
+    if ( params.skip_preprocessing_input ) {
+        println('skipping pre-processing')
```

Comment:
There is a `log.info "Skipping pre-processing"` we could use, it's not documented in Nextflow but I've seen it in other pipelines.

Reply:
maybe later when we change logging for whole pipeline and remove all `print-s`. It requiers more time and testing